app/test_main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+from unittest import mock
+from app.main import can_access_google_page
+from typing import Callable
+import pytest
+
+
+@pytest.mark.parametrize(
+    "values,result", [
+        pytest.param([True, True], "Accessible",
+                     id="User don't have access with internet and valid url"),
+        pytest.param([True, False], "Not accessible",
+                     id="User has access without valid url"),
+        pytest.param([False, True], "Not accessible",
+                     id="User has access without internet"),
+        pytest.param([False, False], "Not accessible",
+                     id="User has access without internet and valid url"),
```

Comment:
Fix all your messages

Reply:
`returns "Accessible" if url is valid to access the Google home page and it has internet connection, else it returns "Not accessible".`